loader/goroot.go

```diff
@@ -259,6 +262,7 @@ func pathsToOverride(goMinor int, needsSyscallPackage bool) map[string]bool {
 
 	if needsSyscallPackage {
 		paths["syscall/"] = true // include syscall/js
+		paths["syscall/wasi/"] = false
```

Comment:
Is syscall/wasi the right place for generated WASI bindings?

Reply:
I wonder if we should move them to `syscall/internal` ?

Reply:
That works. The versioned package generation would ensure generating updates for 0.2.1 etc would not collide with existing code.

Reply:
Moved to `internal/wasi`